docs/source/en/quantization/torchao.md

```diff
@@ -278,6 +279,71 @@ print(tokenizer.decode(output[0], skip_special_tokens=True))
 </hfoption>
 </hfoptions>
 
+### Intel XPU
+<hfoptions id="examples-Intel-XPU">
+<hfoption id="int8-dynamic-and-weight-only">
```

Comment:
better to use the full name "int8_dynamic_activation_int8_weight"

Reply:
yes, i realized it, yet from user perspective, i may think it's confusing, it's not a weight-only quantization but tab named as "weight-only", and the name let user guess what is dynamic(which is actually activation). I will think use its full name will make users clear, @stevhliu, what's your insight in this? Thx.

Reply:
Agree that its better to be clearer upfront

Reply:
fixed